VideoLocker/res/values/strings.xml

```diff
@@ -183,7 +183,7 @@
     <!-- Label for the menu popup for viewing the course on the web -->
     <string name="course_view_on_web">View on Web</string>
     <!-- Label for the menu popup for changing mode to Videos Only/Full Course -->
-    <string name="course_change_mode">Mode picker</string>
+    <string name="course_change_mode">Course mode picker</string>
```

Comment:
@catong  Thoughts on the string for for a11y screen readers
<img width="272" alt="screen shot 2016-08-12 at 7 32 24 am" src="https://cloud.githubusercontent.com/assets/7101723/17621103/1acd5afc-605f-11e6-80ae-0f6ca04dca05.png">